config/manali/cuda_visible_devices.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
This is the same as `balfin/cuda_visible_devices.sh`. right? We could eventually have a scripts folder and not repeat the file.

Reply:
In the case of balfrin+manali, the gpus order is the same:
```
GPU0    48-63,112-127 numa3 gpu=0:03:00.0
GPU1    32-47,96-111  numa2 gpu=0:41:00.0
GPU2    16-31,80-95   numa1 gpu=0:82:00.0
GPU3    0-15,64-79    numa0 gpu=0:C1:00.0
```
so I could use a shared file but it could be different on other gpu systems hence I would prefer to keep it like that for now. We will probably use a shared file later.